### ISSUE TITLE
How to render a mesh to  Silhouette?

### ISSUE THREAD

**Issue by uestcjackey** [tensorflow/graphics#603]
I'm interesting about use a rendered sihouette or texture to cal loss for 3D construction, but it seems  that the tf-graphics dont have these functions. Can you pls give any advice or do you have any plan to develop this? Thanks.

**Comment by solitaryangler** [comment]
Hi, I'm also looking for this solution. Did you find anything relevant?

Any help will be appreciated. 

Developers?

Thank you.

**Comment by uestcjackey** [comment]
> Hi, I'm also looking for this solution. Did you find anything relevant?
> 
> Any help will be appreciated.
> 
> Developers?
> 
> Thank you.

I found some project developed by Pytorch3D, which has differential render, I'm not sure how to do this in tensorflow

**Comment by solitaryangler** [comment]
Thanks @uestcjackey , I also considered PyTorch3D but finally selected NVidia's Kaolin.
